chipsec/helper/rwe/__init__.py

```diff
@@ -20,7 +20,7 @@
 #
 
 import platform, os
-if "windows" == platform.system().lower() and os.environ.has_key("USERWE"):
+if "windows" == platform.system().lower() and (os.path.isfile(os.path.join("C:\\Windows\\System32\\drivers\\RwDrv.sys")) or os.path.isfile(os.path.join(os.getcwd(), "win7_amd64\RwDrv.sys"))):

```

Comment:
Also, I'm not sure it will work when chipsec is built with py2exe. The sys file is not with the rwehelper package. Should probably use "os.path.join( chipsec.file.get_main_dir(), "chipsec", "helper", "rwe", self.win_ver, DRIVER_FILE_NAME )" instead just like rwehelper.py